pkg/kubelet/kubelet.go

```diff
@@ -2967,8 +3015,18 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod, podStatus *kubecontaine
 			// If a resize is in progress, make sure the cache has the correct state in case the Kubelet restarted.
 			kl.statusManager.SetPodResizeInProgressCondition(pod.UID, "", "", false)
 		} else {
+			// podResizeConditionsOriginalLen is used to save original resize condition
+			podResizeConditionsOriginalLen := len(kl.statusManager.GetPodResizeConditions(pod.UID))
```

Comment:
We are holding off on merging https://github.com/kubernetes/kubernetes/pull/131157 for the time being, meaning I wouldn't expect the code that backfills the status manager conditions from there to be merged any time soon. If we want to move forward here, can you copy the condition backfilling into this PR? Basically [this section](https://github.com/kubernetes/kubernetes/pull/131157/files#diff-67dd9e8f3ee257072765326cb4f242852554a2c0753563fa51e292c0a63a7b94R2699)

(This is a follow-up to my other comment: https://github.com/kubernetes/kubernetes/pull/130387#discussion_r2066949117)